app/lib/url.rb

```diff
@@ -70,15 +70,4 @@ def self.local_image(image_name, host: nil)
   def self.organization(organization)
     url(organization.slug)
   end
-
-  # Ensures we don't consider an empty referer
-  #
-  # @param referer [String] the unsanitized referer
-  # @example A safe referer
-  #  sanitized_referer("/some/path") #=> "/some/path"
-  # @example An empty string
-  #  sanitized_referer("") #=> nil
-  def self.sanitized_referer(referer)
```

Comment:
If we revert the suggestion we do need this method because it's called from `application_helper.rb` and `pages_controller.rb `